src/box/lua/config/init.lua

```diff
@@ -578,6 +578,27 @@ function methods.instances(self)
     return res
 end
 
+function methods.instance_uri(self, uri_type, opts)
+    selfcheck(self, 'instance_uri')
```

Comment:
An attempt to access this method before a startup is finished would lead to attempt to index `self._configdata_applied`, which is `nil`. I would call `initcheck()` to raise a proper error in the case.

Reply:
Thanks, fixed.

Reply:
I don't think that it is crucial.